a11y-dialog.js

```diff
@@ -130,11 +131,13 @@
       this.dialog.showModal(event instanceof Event ? void 0 : event);
     } else {
       this.dialog.setAttribute('open', '');
-      this.container.removeAttribute('aria-hidden');
 
       // Iterate over the targets to disable them by setting their `aria-hidden`
-      // attribute to `true`
+      // attribute to `true` and, if present, storing the current value of `aria-hidden`
       this._targets.forEach(function(target) {
+        if (target.getAttribute('aria-hidden')) {
```

Comment:
Shall we use `hasAttribute` here?

Reply:
don't think it makes a functional difference, but sure